modules/react/layout/stories/Grid.stories.mdx

```diff
@@ -0,0 +1,133 @@
+two-dimensional layouts (rows and columns) with
+[CSS Grid](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids).
+
+If you want to see how Flexbox and Grid can
+[work together](https://css-tricks.com/css-grid-replace-flexbox/).
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+Below is a basic layout built with `Grid`. We created four components: `Header`, `SideBar`,
+`BodyContent` and `Footer`. We also used those names for their respective `gridArea` props. Then we
+can use those names in the parent component, `UIExample`, to organize the UI however we want. This
+allows the parent container to control how its children render. For page layouts, Workday typically
+uses a 12 column grid. In the example below, the Sidebar is 3 columns and the Body Content is 9
+columns.
+
+> Note: You can learn more about CSS Grid's `gridArea` and `gridTemplateAreas` >
+> [here](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-areas).
+
+<ExampleCodeBlock code={UIExample} />
+
+### Using Grid Items
+
+In the example above we nested `Grid` components to created our layout, and we controlled the layout
+structure from the top-level `Grid` container. That's a perfectly valid approach. However, we can
+also use `Grid.Item` components to allow child cells to have more control. While any direct child of
+a `Grid` component is implicitly a grid item, `Grid.Item` provides special CSS Grid Item style props
+that allow you to have more control over how and where each item renders.
+
+To demonstrate this behavior, the example below has a `Grid` container with nine cells. The eight
+`soap500` cells are `Grid` components, and the `peach300` cell is a `Grid.Item`. We can use the
+`Grid.Item` style props `gridRowStart` and `gridColumnStart` to manipulate where the cell renders.
+You can use the `Row` and `Column` buttons to manipulate these props and see the `Grid.Item`'s
+position adjust accordingly.
+
+> Note: This example is solely intended to demonstrate the `Grid.Item`'s functionality and is NOT
+> considered an accessibility best practice. Visually reordering content does not change the order
+> it is read by a screen reader or the tab order. You can learn more about CSS Grid Layout and
+> Accessibility
+> [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/CSS_Grid_Layout_and_Accessibility).
+
+<ExampleCodeBlock code={GridLayoutInteractive} />
+
+In the second example below, we have a `Grid` container. It contains two rows with one having seven
+elements and the other has two elements. This example is to show how a user can use the
+`<Grid.Item/>` component to wrap a `<Grid/>` component. This would allow a user to have access to
+the child props of `<Grid.Item/>` to place a layout where needed. The row with elements 3 - 7 is put
+before the 1 & 2 elements since that layout is wrapped by the `<Grid.Item/>` component.
```

Comment:
```suggestion
`<Grid.Item/>` component to wrap a `Grid` component. This would allow a user to have access to
the child props of `Grid.Item` to place a layout where needed. The row with elements 3 - 7 is put
before the 1 & 2 elements since that layout is wrapped by the `Grid.Item` component.
```

Reply:
Typically, in text blocks we use inline code blocks with capitalized text for component names in our docs: `Grid` and `Grid.Item` without adding any markup brackets.